specification.markdown

```diff
@@ -315,37 +315,47 @@ The `name` attribute is REQUIRED.
 
 #### path
 
-The path defines the allowed storage locations of the `EBML Element` within an `EBML Document`. This path MUST be defined with the full hierarchy of `EBML Elements` separated with a `/`. The top `EBML Element` in the path hierarchy being the first in the value. The syntax of the `path` attribute is defined using this Augmented Backus-Naur Form (ABNF) [@!RFC5234] notation:
+The path defines the allowed storage locations of the `EBML Element` within an `EBML Document`. This path MUST be defined with the full hierarchy of `EBML Elements` separated with a `/`. The top `EBML Element` in the path hierarchy being the first in the value. The syntax of the `path` attribute is defined using this Augmented Backus-Naur Form (ABNF) [@!RFC5234] with the case sensitive update [@!RFC7405] notation:
 
 The `path` attribute is REQUIRED.
 
 ```
-EBMLFullPath          = [EBMLElementOccurence] EBMLReferencePath
-EBMLReferencePath     = EBMLParentPath EBMLElementPath
-EBMLParentPath        = *EBMLPathAtom / AllWildcard
-EBMLElementPath       = EBMLPathAtom / EBMLPathAtomRecursive
-EBMLPathAtom          = PathDelimiter EBMLAtomName
-EBMLPathAtomRecursive = "(1*" EBMLPathAtom ")"
-EBMLAtomName          = 1*EBMLNameChar / AllWildcard
-EBMLNameChar          = [A-Z] / [a-z] / [0-9] / "-" / "."
-AllWildcard           = "any"
-PathDelimiter         = "\"
-MinOccurence          = [0-9]+
-MaxOccurence          = [0-9]+
-EBMLElementOccurence  = [MinOccurence] "*" [MaxOccurence]
+EBMLFullPath            = EBMLElementOccurence "(" EBMLReferencePath ")"
+EBMLReferencePath       = [EBMLParentPath] EBMLElementPath
+EBMLParentPath          = EBMLFixedParent EBMLLastParent
+EBMLFixedParent         = *(EBMLPathAtom)
+EBMLElementPath         = EBMLPathAtom / EBMLPathAtomRecursive
+EBMLPathAtom            = PathDelimiter EBMLAtomName
+EBMLPathAtomRecursive   = "1*(" EBMLPathAtom ")"
+EBMLLastParent          = EBMLPathAtom / EBMLVariableParent
+EBMLVariableParent      = VariableParentOccurence "(\any)"
+EBMLAtomName            = 1*(EBMLNameChar)
+EBMLNameChar            = ALPHA / DIGIT / "-" / "."
+PathDelimiter           = "\"
+AllWildcard             = %s"any"
+EBMLElementOccurence    = [EBMLMinOccurence] "*" [EBMLMaxOccurence]
+EBMLMinOccurence        = 1*DIGIT
+EBMLMaxOccurence        = 1*DIGIT
+VariableParentOccurence = [PathMinOccurence] "*" [PathMaxOccurence]
+PathMinOccurence        = 1*DIGIT
+PathMaxOccurence        = 1*DIGIT
 ```
 
-The `"*"`, `"("` and `")"` symbols MUST be interpreted as they are defined in the ABNF. The `EBMLAtomName` MUST NOT use the reserved word `"any"` corresponding to the `AllWildcard` value.
+The `"*"`, `"("` and `")"` symbols MUST be interpreted as they are defined in the ABNF.
```

Comment:
Why is the warning about the reserved word `any` removed. If an `EBML DocType` defined `any` as an `Element Name` would there be a confusion in interpretting the path.